code/bml_loader.js

```diff
@@ -0,0 +1,20 @@
+function loadBookmarklet(kbdid, langid) {
+  var loader = () => window['keyman'].addKeyboards(kbdid + "@" + langid);
+
+  if(window['keyman'] instanceof Promise) {
+    window['keyman'].then(loader);
+  } else if(window['keyman'] instanceof Object) {
+    loader();
+  } else {
+    window['keyman'] = new Promise((resolve, reject) => {
+      try {
+        var e = document.createElement('script');
+        e.type='text/javascript';
+        e.onload = resolve;
+        e.onerror = reject;
+        e.src='https://r.keymanweb.com/code/bml20.php?langid=' + langid + '&keyboard=' + kbdid;
```

Comment:
```suggestion
        e.src='https://r.keymanweb.com/code/bml20.php?langid=' + encodeURIComponent(langid) + '&keyboard=' + encodeURIComponent(kbdid);
```

Reply:
Also curious why you dropped `&lang=...` for the language name -- this is visible in the keyboard switcher ui.

Reply:
Because it's not set by the bookmarklet; it's set by the bookmarklet's cloud request that retrieves the keyboard.

Compare:  the corresponding code removed from keyman.com's current implementation:

https://github.com/keymanapp/keyman.com/blob/3df2cfd3b2c4a421ed4393e9d0af96bea18f5fa9/bookmarklet/install-bookmarklet.js#L3-L11

```javascript
var code =
    "javascript:void((function(){try{var%20e=document.createElement('script');e.type='text/javascript';"+
    "e.src='"+bookmarkletParameters.resourceBase+"/code/bml20.php"+
    "?langid="+encodeURIComponent(lang.id)+
    "&amp;keyboard="+encodeURIComponent(kbd.id)+
    "';document.body.appendChild(e);}catch(v){}})())";
```

The keyboards load perfectly fine in the language picker when using the bookmarklet generated by this code:

![image](https://user-images.githubusercontent.com/25213402/199630916-10716cf7-8327-4a55-913e-1584680a8226.png)

Reply:
For additional reference, here's how the base `bml20.php` call loads the keyboard:

https://github.com/keymanapp/r.keymanweb.com/blob/f1e60ac3261025af6426740e4b088c2b5c82945b/code/bml.php#L145-L153

It doesn't even glance at `&lang`.

Reply:
Righto. So it's just legacy cruft. Good stuff :grin: